pkg/registry/certificates.go

```diff
@@ -14,20 +14,16 @@ const (
 	insecureFile       = "insecure"
 )
 
-func GetManagementClusterCertificate() (certificates *x509.CertPool, err error) {
-	return GetClusterCertificate(os.Getenv("CLUSTER_NAME"))
-}
-
 func GetRegistryInsecure(clusterName string) bool {
-	caFile := path.Join(registryConfigPath, clusterName, insecureFile)
+	caFile := path.Join(registryConfigPath, clusterName+"_"+insecureFile)
 	if _, err := os.Stat(caFile); err != nil {
 		return false
 	}
 	return true
 }
 
 func GetClusterCertificateFileName(clusterName string) string {
-	caFile := path.Join(registryConfigPath, clusterName, certFile)
+	caFile := path.Join(registryConfigPath, clusterName+"_"+certFile)
```

Comment:
Did the secret not handle putting the file in a "directory"? That would have been nice I think, but oh well.

Reply:
`Secret "registry-mirror-cred" is invalid: [data[vni-mgmt/ca.crt]: Invalid value: "vni-mgmt/ca.crt": a valid config key must consist of alphanumeric characters, '-', '_' or '.' (e.g. 'key.name',  or 'KEY_NAME',  or 'key-name', regex used for validation is '[-._a-zA-Z0-9]+'), data[vni-mgmt/config.json]: Invalid value: "vni-mgmt/config.json": a valid config key must consist of alphanumeric characters, '-', '_' or '.' (e.g. 'key.name',  or 'KEY_NAME',  or 'key-name', regex used for validation is '[-._a-zA-Z0-9]+'), data[vni-mgmt/insecure]: Invalid value: "vni-mgmt/insecure": a valid config key must consist of alphanumeric characters, '-', '_' or '.' (e.g. 'key.name',  or 'KEY_NAME',  or 'key-name', regex used for validation is '[-._a-zA-Z0-9]+')]`